### ISSUE TITLE
Remote Access Authorization Error when trying to connect to Salesforce Inspector Reloadded

### ISSUE THREAD

**Issue by sumanth86** [tprouvot/Salesforce-Inspector-reloaded#449]
**Make sure to read the troubleshooting section before creating an issue**

- [ ] I've read the [common issues](https://tprouvot.github.io/Salesforce-Inspector-reloaded/troubleshooting/) and the one I'm about to create is not one of those documented.

**Describe the bug**
We have installed the Salesforce Inspector Connected App and added the required profiles who need to use them.
while it works for system admin, it does not work for the non system-admin profiles.
Details are shown in the screenshot

**To Reproduce**
1. Go to Setup as Non-System Admin
2. Click on Salesforce Inspector and click Generate Token
3. See error - "Remote Access Authorization'"

**Expected behavior**
As soon as we launch Salesforce inspector and click generate token, it should start working. 

**Screenshots**
 This error is seen for a non system admin, while it works for system admin
![image](https://github.com/tprouvot/Salesforce-Inspector-reloaded/assets/6827068/632054bb-a0b8-4079-9563-dadec560d379)

**Desktop (please complete the following information):**
 - OS:  Windows
 - Browser: Chrome
 - Version: Latest Version

**Additional context**

**Comment by tprouvot** [comment]
Hi @sumanth86 ,

I won't be able to check for this bug until next week, sorry for the delay.
I'll come back to you if I can reproduce the error

**Comment by sumanth86** [comment]
Hi @tprouvot  Any update on this, we want to launch Salesforce Inspector Reloadded App to be used across the organization and with this issue it will be a blocker for us.

**Comment by tprouvot** [comment]
Hi @sumanth86 ,
Currently in holidays I don't have access to a computer, I'll try to check on Friday if I can and beginning of next week in the worst case scenario.
Did you checked if you could reproduce this error in an empty scratch org / dev org?

**Comment by sumanth86** [comment]
I was able to reproduce this in our Sandbox, I can try out in Dev Org.
Sandbox as refreshed recently - FYI

**Comment by tprouvot** [comment]
Yes the goal was to try on an empty environment without any of your congratulations.

**Comment by sumanth86** [comment]
I suspect its v60.0 issue that is causing Salesforce Inspector issue with users non-admins who are refreshed from production. @tprouvot

**Comment by tprouvot** [comment]
I'll try for the auth token, for the login as I think it's a permission issue (user needs login as permissions to be able to use the button)

**Comment by sumanth86** [comment]
what permissions should we provide at a profile level to launch the Salesforce Inspector?
why does it work for system admin and doesnt work for non-admin?
